DuckDuckGo/Bookmarks/Model/BookmarkDragDropManager.swift

```diff
@@ -49,6 +49,11 @@ final class BookmarkDragDropManager {
         // If the dragged values contain both folders and bookmarks, only validate the move if all objects can be moved.
         case (true, true), (true, nil), (nil, true):
             return .move
+        case (nil, nil):
```

Comment:
can we instead add `case (false, _), (_, false): return .none` with the comment above and use the `default` case checking the URL/String value for better readability?

Reply:
sure!